src/pybind/cephfs/cephfs.pyx

```diff
@@ -197,17 +197,18 @@ class WouldBlock(Error):
 class OutOfRange(Error):
     pass
 
+""" FreeBSD does not have ENODATA """
```

Comment:
@tchaikov 
My bad, I was thinking python comment block, but then reduced the amount of text.
And a `#` will do.
Will rebase.